### PR TITLE
Replace content with the services CLI output

### PR DESCRIPTION
**Pull request by Siegrift**
The beacon reader starter project is supposed to be generate by https://github.com/api3dao/services repository, so this PR did just that. It first remove all files and then used the services CLI based on the following PR https://github.com/api3dao/services/pull/7 to initialize this project "from scratch"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,79 +2,126 @@
...
 ```sh
 npm install
+# or
+yarn install
 ```
 
-Now depending on the network you are using you will need to do the following:
-
-### Default network
+### Temporary instructions until servies is properly published
```
Comment (dcroote) [inline README.md]: Typo
```suggestion
### Temporary instructions until services is properly published
```

File: README.md [hunk 2]
```diff
@@ -2,79 +2,126 @@
...
+4. A script to read the beacon value from the deployed contract.
 
 ## Instructions
 
-1. Clone this repo
-1. Run the following to install the dependencies
+Run the following to install the dependencies
 
 ```sh
 npm install
```
Comment (dcroote) [inline README.md:27]: On a fresh `git clone`, `npm install` didn't work for me either, but `yarn install` did.

Consequently / obviously, none of the subsequent `npm` commands worked.

File: README.md [hunk 3]
```diff
@@ -2,79 +2,126 @@
...
-This project is composed of the following:
-
-1. A test that uses a mocked RrpBeaconServer to simulate reading a beacon value from a smart contract.
-1. A script to deploy the BeaconReaderExample contract to any network. This script will set the address of the RrpBeaconServer contract deployed on the selected network and after the script finishes it will display the address of the contract deployed but also save this address to `deployments` directory on the root of this repo (except for hardhat network since this network is started and stop just for the duration of the script execution).
-1. A script to read the beacon value from the deployed contract. If selected network is not `hardhat` or `localhost` then this address must be then whitelisted by the RrpBeaconServer manager account prior to reading the beacon value.
+This project was created by the `create-beacon-reader-app` CLI from the
+[services repo](https://github.com/api3dao/services). This project is composed
+of the following:
+
+1. A test that uses a mocked `RrpBeaconServer` contract to simulate reading a
```
Comment (dcroote) [inline README.md:9]: I would add:
```
on `localhost`
```
to after "contract"

File: README.md [hunk 4]
```diff
@@ -2,79 +2,126 @@
...
+
 ```sh
-npx hardhat run scripts/deploy.js --network polygon-mumbai
-npx hardhat run scripts/read-beacon.js --network polygon-mumbai
+npm run deploy --network polygon-mumbai
+npm run whitelist-reader --network polygon-mumbai
+npm run read-beacon --network polygon-mumbai
+# or
+yarn deploy --network polygon-mumbai
+yarn whitelist-reader --network polygon-mumbai
```
Comment (dcroote) [inline README.md:115]: These 3 steps (`yarn`) were successful for me on `rinkeby`, not on `polygon-mumbai`:

The `deploy` step worked, but `whitelist-reader` failed with the below error. 
May be the naming issue: `polygonMumbai` within https://github.com/api3dao/utility-contracts/tree/main/SelfServeRrpBeaconServerWhitelister/deployments

```sh
> yarn whitelist-reader --network polygon-mumbai
yarn run v1.22.11
$ hardhat run scripts/whitelist-reader.js --network polygon-mumbai
Error: No whitelister contract found for chain: polygon-mumbai
    at /home/derek/api3/beacon-reader-example/node_modules/@api3/services/src/index.ts:75:48
    at step (/home/derek/api3/beacon-reader-example/node_modules/@api3/services/dist/index.js:33:23)
    at Object.next (/home/derek/api3/beacon-reader-example/node_modules/@api3/services/dist/index.js:14:53)
    at /home/derek/api3/beacon-reader-example/node_modules/@api3/services/dist/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/derek/api3/beacon-reader-example/node_modules/@api3/services/dist/index.js:4:12)
    at whitelistBeaconReader (/home/derek/api3/beacon-reader-example/node_modules/@api3/services/dist/index.js:70:12)
    at main (/home/derek/api3/beacon-reader-example/scripts/whitelist-reader.js:34:24)
    at Object.<anonymous> (/home/derek/api3/beacon-reader-example/scripts/whitelist-reader.js:54:1)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Review by dcroote — COMMENTED** [review]
`localhost` and `rinkeby` flows worked, but only with `yarn`, and `polygon-mumbai` did not work.

File: README.md (hunk 2 above)
Comment (vponline) [reply README.md:27]: Same for me when trying to install with npm and it seems to be a problem with installing the `api3dao/services` dependency in `package.json`. Perhaps will be resolved after the package is published.

File: README.md (hunk 1 above)
Comment (Siegrift) [reply README.md]: This will get removed hopefully today, but I'll fix it now anyway :D

File: README.md (hunk 3 above)
Comment (Siegrift) [reply README.md:9]: You can actually run those test on non localhost network as well (if you specify `--network ropsten` after `yarn test`).

File: README.md (hunk 4 above)
Comment (Siegrift) [reply README.md:115]: Yeah, I thought so too. Will update this.

File: README.md (hunk 2 above)
Comment (Siegrift) [reply README.md:27]: Yeah, this thread suggests that as well https://stackoverflow.com/questions/56118038/npm-err-premature-close-when-using-npm-install.

I will try to wrap up services repo work today and than we can try again (after Burak publishes the pckgs)

**Review by vponline — COMMENTED** [review]
It could make sense to add a note into the README that the example whitelist script is for the `eth_usd` beacon?

File: README.md (hunk 4 above)
Comment (Siegrift) [reply README.md:115]: Should be fixed by https://github.com/api3dao/utility-contracts/pull/2

**Comment by Siegrift** [comment]
`npm i` should work now. I am merging this. Feel free to test.

**Comment by vponline** [comment]
`npm i` works now, but I think the npm script examples in the README need to be with a `=` in the arguments, e.g. `--network={network}`. 

I'm also getting an error when trying to deploy with `npm run deploy --network=ropsten`:

> Error: Deployment file for hardhat not found
>     at getServiceData (/home/vp/projects/beacon-reader-npm/node_modules/@api3/services/src/index.ts:30:11)
>     at main (/home/vp/projects/beacon-reader-npm/scripts/deploy.js:35:25)

**Comment by Siegrift** [comment]
@vponline You are right, it's likely due to how npm parses and passes arguments to the invoked process...

I've created https://api3dao.atlassian.net/browse/BEC-146 for it

**Pull request merged**